src/encoded/static/components/filegallery.js

```diff
@@ -1776,14 +1831,35 @@ export function assembleGraph(files, highlightedFiles, dataset, options, loggedI
                 type: 'Coalesced',
                 shape: 'stack',
                 cornerRadius: 16,
-                contributing: groupHash,
                 ref: coalescingGroup,
                 displayDecoration: true,
                 decorationClass: infoNode && infoNode.id === fileNodeId ? 'decoration--active' : '',
             });
         }
     });
 
+    // Add coalesced reads nodes to the graph.
+    Object.keys(readsCoalescingGroups).forEach((groupHash) => {
+        // Get all file @ids in the coalescing group.
+        const coalescingGroupFiles = readsCoalescingGroups[groupHash];
+
+        // Get first file object and get its biological_replicate.
+        const replicateNode = jsonGraph.getNode(`rep:${coalescingGroupFiles[0].biological_replicates[0]}`);
+        const fileNodeId = `coalesced:${groupHash}`;
```

Comment:
Seems like Id should never have words in a dictionary in "coalesced" in it, by intent